docs/release/release_0_6_0.md

```diff
@@ -179,25 +186,37 @@ You'll notice the main napari GUI is subtly (or not so subtly) different in
 - If you select multiple layers in the layer list, you can now see the status
   display of all the selected layers in the status bar
   ([#7673](https://github.com/napari/napari/pull/7673))
+- If you switch from 2D to 3D view and back again, your 3D viewing angle will
+  be preserved ([#7765](https://github.com/napari/napari/pull/7765))
 
 ### Other stuff
 
-For developers: napari now depends on Python 3.10+ and Pydantic v2.2.
+For developers: napari now depends on Python 3.10+
+([#7603](https://github.com/napari/napari/pull/7603) and Pydantic v2.2
+([#7589](https://github.com/napari/napari/pull/7589)).
 
 We've supported both pydantic 1 and 2 since 0.4.19, but we're now ready to take
 advantage of performance and API improvements in Pydantic 2. If your library
 depends on Pydantic 1.x, now would be a good time to upgrade, or it will not be
 compatible with napari going forward.
-([#7589](https://github.com/napari/napari/pull/7589))
 
-- Flip z axis on 3D camera to default to right-handed frame (#7488 redux) ([#7554](https://github.com/napari/napari/pull/7554))
-- Update configuration to drop python 3.9 and add python 3.13 ([#7603](https://github.com/napari/napari/pull/7603))
-- Change deprecation to 0.7.0 for private attribute access from PublicOnlyProxy ([#7730](https://github.com/napari/napari/pull/7730))
-- Maintain 3D camera angles when switching between 2D and 3D views ([#7765](https://github.com/napari/napari/pull/7765))
+
+If you were dreading 0.6.0 because you were relying on `_qt_viewer` features,
+worry not: we have again postponed that deprecation while we add the required
+APIs. 😅 ([#7730](https://github.com/napari/napari/pull/7730))
+
+Also, if you've been wanting to contribute that doc fix but found the process
+daunting, it's now easier than ever, because our default documentation preview
+build is now much faster! You can propose changes from the GitHub UI and see
+the rendered results in only two minutes!
+([napari/docs#669](https://github.com/napari/docs/pull/669))
+
+- Add a custom log handler and GUI viewer with filters ([#6900](https://github.com/napari/napari/pull/6900))
```

Comment:
This is duplicated below - is that intentional?

Reply:
Ah nice, it's semi-intentional in that it's in the highlights but I didn't write a highlight blurb. (All highlighted PRs also appear in their relevant sections.)

Thanks for spotting it!